recipes/install_redis.rb

```diff
@@ -0,0 +1,40 @@
+# Cookbook Name:: abiquo
+# Recipe:: install_redis
+#
+# Copyright 2014, Abiquo
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include_recipe 'redisio'
+
+redis_settings =  { 
+    'address'                   => '0.0.0.0', 
+    'user'                      => 'redis',
+    'configdir'                 => '/etc',
+    'datadir'                   => '/var/lib/redis',
+    'maxclients'                => 10000,
+    'clientoutputbufferlimit'   => [
+        ['normal', '0', '0', '0'],
+        ['slave', '256mb', '64mb', '60'],
+        ['pubsub', '32mb', '8mb', '60']
+    ]
+}
+
+redisio_configure "redis" do
+  version nil
+  default_settings redis_settings
+  servers [{'port' => '6379'}]
+  base_piddir node['redisio']['base_piddir']
+end
+
+include_recipe 'redisio::enable'
```

Comment:
Should we provide a couple attributes to make the listen address and port configurable?